chrome/android/chrome_java_sources.gni

```diff
@@ -1830,6 +1830,7 @@ chrome_java_sources = [
 
 chrome_java_sources += [
   "java/src/org/chromium/chrome/browser/AlwaysOnTopService.java",
+  "java/src/org/chromium/chrome/browser/CastanetsSettings.java",
```

Comment:
Should this be added only when Castanets flag is on?

Reply:
You're right but we should use dynamic loading for solving this problem.
I'm going to refactor later... :)